### PR TITLE
Add acknowledgement doc section

### PR DESCRIPTION
**Pull request by MaxThevenet**

### CODE REVIEW

File: Docs/source/acknowledge.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Acknowledge WarpX
+=================
+
+Please acknowledge the role that WarpX played in your research by adding the following sentence to your publications, it helps contributors keep track of users and promote the project:
```
Comment (LDAmorim) [inline Docs/source/acknowledge.rst]: Thanks for this PR @MaxThevenet !
It is very useful to have a prepared sentence.
But I would not phrase it like that "contributors (want to) keep track of users". Maybe more "community" centered would be nicer.

File: Docs/source/acknowledge.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  \usepackage{hyperref}
+  This research used the open-source particle-in-cell code WarpX \url{https://github.com/ECP-WarpX/WarpX}, primarily funded by the US DOE Exascale Computing Project. We acknowledge all WarpX contributors.
+
+**Main WarpX reference:**
+
+.. raw:: html
+
+    <div id="ref-VayAAC2010">
+
+Vay, J -. L, C G R Geddes, C Benedetti, D L Bruhwiler, E Cormier-Michel, B M Cowan, J R Cary, and D P Grote. 2010. “Modeling Laser Wakefield Accelerators in A Lorentz Boosted Frame.” *Aip Conference Proceedings* 1299: 244–49. https://doi.org/10.1063/1.3520322.
```
Comment (ax3l) [inline Docs/source/acknowledge.rst]: Isn't the last EAAC paper (see readme) our current go-to reference? :)

**Comment by LDAmorim** [comment]
Maybe we can add a note or suggestion to make a QR code with WarpX tools for posters too :)

File: Docs/source/acknowledge.rst (hunk 1 above)
Comment (MaxThevenet) [reply Docs/source/acknowledge.rst]: Oh yes, thanks, I'll improve that!

File: Docs/source/acknowledge.rst (hunk 2 above)
Comment (MaxThevenet) [reply Docs/source/acknowledge.rst]: Oops, I copy-pasted the reference just for formatting, and forgot to replace it with the actual paper..

**Pull request merged**